### ISSUE TITLE
Demo App crash - Fatal error: Index out of range: file 

### ISSUE THREAD

**Issue by bizready** [aws/amazon-chime-sdk-ios#69]
**Describe the bug**
I was trying to run the demo code. 

**To Reproduce**
1. Using Xcode, run the code and select Emulator iPhone (I selected, iPhone8, 11)
2. App loaded on my the emulator 
3. Enter meeting ID and, Name and start the session by clicking on phone button
4. App crashed and see the console error below

**Expected behavior**
The meeting page is shown briefly and then it crashes

**Logs**
```
2020-07-10 16:40:57.101771-0400 AmazonChimeSDKDemo[4441:14488516] [INFO] MeetingViewController - Audio successfully started. Reconnecting: false
Fatal error: Index out of range: file /Library/Caches/com.apple.xbs/Sources/swiftlang/swiftlang-1103.2.25.8/swift/stdlib/public/core/ContiguousArrayBuffer.swift, line 444
2020-07-10 16:40:57.246875-0400 AmazonChimeSDKDemo[4441:14488516] Fatal error: Index out of range: file /Library/Caches/com.apple.xbs/Sources/swiftlang/swiftlang-1103.2.25.8/swift/stdlib/public/core/ContiguousArrayBuffer.swift, line 444
```

**Test environment Info (please complete the following information):**
 - Device: [e.g. Emulator iPhone8, Emulator iPhone11] on XCode 11.5
 - Version AmazonChimeSDK: [0.7.1]
 - Version AmazonChimeSDKMedia: [0.5.1]

**Comment by wanghaor-amzn** [comment]
Hi, @bizready how often do you encounter this issue? Do you experience the same issue on physical devices? Also, have you ever had such an issue in previous version of AmazonChimeSDK?

I am using the test devices and packages you mentioned in **Test environment Info** section above but so far I wasn't able to reproduce the issue. I hope you can provide us more details of it.

Thanks!

**Comment by bizready** [comment]
This is the first time I am using the demo code to experiment. I have not tried it on my iPhone device as it has some problems running on the physical device. Will update if I can reproduce it again.

**Comment by wanghaor-amzn** [comment]
Hi, @bizready 
We just released AmazonChimeSDK v0.8.0 and AmazonChimeSDKMedia v0.5.2.
Please let us know if you still experience crash for demo app with the latest binaries.
Thanks!

**Comment by tianyux-amz** [comment]
Closing the issue as we did not hear back from the builder. Please reopen the issue if you have more questions.
